### PR TITLE
Finish List of Interviwers

### PR DESCRIPTION
**Pull request by CHILL99**
Finish List of Interviwers

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	before_action :authenticate_user!
+	before_action :check_admin
+
+	def index
+    @users = User.all
+  end
+
+	private
+
+		def check_admin
```
Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: problems with indentation

File: app/views/admin/users/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+%h1 Interviewers
```
Comment (Kmalynss) [inline app/views/admin/users/index.html.haml]: We can not have part of commits in haml and others in erb.
And haml isn't best option.

**Comment by Kmalynss** [comment]
good branch naming

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -13,4 +13,4 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
-//= require_tree .
+//= require_tree .
```
Comment (serhiym) [inline app/assets/javascripts/application.js]: it's minor but do not remove last new line ) to avoid unnecessary changes

File: app/controllers/admin/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	  	if current_user.admin?
```
Comment (serhiym) [inline app/controllers/admin/users_controller.rb]:
```
def check_admin
  if current_user.admin?
end
```
even with proper indentation this method looks strange))

Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: agree
